WcaOnRails/app/webpacker/components/GenderSelector/GenderSelector.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import _ from 'lodash';
+import { Form } from 'semantic-ui-react';
+
+import genders from '../../lib/wca-data/genders.js.erb';
+
+const genderOptions = _.map(genders.byId, (gender) => ({
+  key: gender.id,
+  text: gender.name,
+  value: gender.id,
+}));
```

Comment:
Since the rationale for moving or not the options is the same as for `countryOptions` I'd suggest to close this conversation and discuss on the other one :)